op-deployer/pkg/deployer/manage/flags.go

```diff
@@ -56,6 +56,18 @@ var (
 		EnvVars: deployer.PrefixEnvVar("DISPUTE_ABSOLUTE_PRESTATE"),
 		Value:   standard.DisputeAbsolutePrestate.Hex(),
 	}
+	DisputeAbsolutePrestateCannonFlag = &cli.StringFlag{
+		Name:    "dispute-absolute-prestate-cannon",
+		Usage:   "The absolute prestate hash for the CANNON dispute game. Defaults to the standard value.",
+		EnvVars: deployer.PrefixEnvVar("DISPUTE_ABSOLUTE_PRESTATE_CANNON"),
+		Value:   standard.DisputeAbsolutePrestate.Hex(),
+	}
+	DisputeAbsolutePrestateCannonKonaFlag = &cli.StringFlag{
+		Name:    "dispute-absolute-prestate-cannon-kona",
+		Usage:   "The absolute prestate hash for the CANNON_KONA dispute game. Defaults to the standard value.",
+		EnvVars: deployer.PrefixEnvVar("DISPUTE_ABSOLUTE_PRESTATE_CANNON_KONA"),
+		Value:   standard.DisputeAbsolutePrestate.Hex(),
+	}
```

Comment:
These haven't actually been added to the migrate command at the bottom of the file.